### DISCUSSION TITLE
Does Aptos have a burn address?

### DISCUSSION THREAD

**Discussion by hariria** [aptos-labs/aptos-developer-discussions#134] [Questions] [answered]
## Solution

The quick answer is no. 

The real answer is a bit nuanced.

Ethereum has a burn address ([0x0](https://etherscan.io/address/0x0000000000000000000000000000000000000000) I believe). No one owns the private key for this address, and thus when tokens are sent there, they are in effect "removed" from the circulating supply. If you want to learn more about how this mechanism is used for transaction fees, you can see [EIP 1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md). 

On Aptos, there is no official burn address. Furthermore, data can be deleted, not just sent to an unclaimable address. If someone wants the state / data associated with a resource to be deletable, the resource should implement a function or capability that allows an individual (typically the owner / creator) to delete the resource. [`0x3::token::burn_by_creator`](https://github.com/aptos-labs/aptos-core/blob/3d4e63c51d50b6dd3d421663089153df5083747a/aptos-move/framework/aptos-token/sources/token.move#L502) is a good example of this.

### What about `burn.apt`?

Some folks have been confused to believe that the account with [ANS name](https://www.aptosnames.com/) `burn.apt` is Aptos' burn address when in fact this is just a normal account. And it's unclear if anyone owns the private keys to this account.

You can see that none of the tokens / resources sent to that account have been burned by looking at the [tokens page for that account on the explorer](https://explorer.aptoslabs.com/account/0xe98dda420f0c22a85de3c95f6245400a6eb52b22caccd690152e5c55633da1af/tokens?network=mainnet&page=1). If they had been burned, these tokens would no longer exist.

**Comment by heliuchuan** [comment, marked as answer]
No

**Comment by BriungRi** [comment]
Technically, if you send tokens to an address that cannot be controlled by a single private key, that address can function as a burn address.

`0x1` is probably the closest example of such an address.
